module/x/gravity/keeper/grpc_query.go

```diff
@@ -221,9 +222,21 @@ func (k Keeper) UnsignedSignerSetTxs(c context.Context, req *types.UnsignedSigne
 		}
 		return false
 	})
+	k.IterateCompletedOutgoingTxsByType(ctx, types.SignerSetTxPrefixByte, func(_ []byte, cotx types.OutgoingTx) bool {
```

Comment:
It looks the orchestrator assumes oldest first, but there is no functional purpose for this other than logging the nonces in increasing order since it loops and signs all of them anyway. Moving the Completed iterator above should make them stay in order as it seems they were before.